### PR TITLE
Update relays_node to latched message

### PR DESCRIPTION
**Pull request by Kotochleb**

### CODE REVIEW

File: panther_power_control/src/relays_node.py [hunk 1]
```diff
@@ -1,39 +1,52 @@
...
         # -------------------------------
 
-        self._motor_on_pub = rospy.Publisher('hardware/motor_on', Bool, queue_size=1)
-        self._e_stop_state_pub = rospy.Publisher('hardware/e_stop', Bool, queue_size=1)
+        self._motor_on_pub = rospy.Publisher(
+            'hardware/motor_on', Bool, queue_size=1, latch=True
+        )
+        self._e_stop_state_pub = rospy.Publisher(
+            'hardware/e_stop', Bool, queue_size=1, latch=True
+        )
```
Comment (KmakD) [inline panther_power_control/src/relays_node.py]: This can be one liners

File: panther_power_control/src/relays_node.py [hunk 2]
```diff
@@ -1,39 +1,52 @@
...
 
         # -------------------------------
         #   Subscribers
         # -------------------------------
 
-        self._cmd_vel_sub = rospy.Subscriber('/cmd_vel', Twist, self._cmd_vel_cb, queue_size=1)
+        self._cmd_vel_sub = rospy.Subscriber(
+            '/cmd_vel', Twist, self._cmd_vel_cb, queue_size=1
+        )
```
Comment (KmakD) [inline panther_power_control/src/relays_node.py]: Same as above

File: panther_power_control/src/relays_node.py [hunk 3]
```diff
@@ -50,44 +63,44 @@ def __init__(self, name) -> None:
...
+    def _cmd_vel_cb(self, *args) -> None:
+        self._cmd_vel_msg_time = rospy.get_time()
 
     def _e_stop_reset_cb(self, req: TriggerRequest) -> TriggerResponse:
-        if time() - self._cmd_vel_msg_last_time <= 2.0:
+        if rospy.get_time() - self._cmd_vel_msg_time <= 2.0:
             return TriggerResponse(
                 False,
-                'E-STOP reset failed, some messages are published on the /cmd_vel topic',
+                'E-STOP reset failed, /cmd_vel is still being published!',
```
Comment (KmakD) [inline panther_power_control/src/relays_node.py]: This sentence sounds a bit weird, you don't publish topic rather messages to a topic. IMO previous was fine.

**Review by KmakD — CHANGES_REQUESTED** [review]

File: panther_power_control/src/relays_node.py (hunk 1 above)
Comment (Kotochleb) [reply panther_power_control/src/relays_node.py]: Black had different idea

File: panther_power_control/src/relays_node.py (hunk 3 above)
Comment (Kotochleb) [reply panther_power_control/src/relays_node.py]: Well. This is message copied from `power_board_node`

File: panther_power_control/src/relays_node.py (hunk 1 above)
Comment (KmakD) [reply panther_power_control/src/relays_node.py]: You sure you have line_length set to 100? My black makes it one line

File: panther_power_control/src/relays_node.py (hunk 3 above)
Comment (KmakD) [reply panther_power_control/src/relays_node.py]: Maybe we should change it in `power_board_node` too

**Review by KmakD — CHANGES_REQUESTED** [review]

**Review by KmakD — APPROVED** [review]

**Review by KmakD — APPROVED** [review]

**Pull request merged**